app/javascript/spotlight/channels/consumer.js

```diff
@@ -0,0 +1,6 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the `bin/rails generate channel` command.
+
+import { createConsumer } from "@rails/actioncable"
```

Comment:
I don't think this file belongs in the engine. It should be in the host application.  By doing that you can remove the `@rollup/plugin-node-resolve`

Reply:
You are suggesting this goes in the install generator?

Reply:
Isn't it already in the actioncabel generator?

Reply:
Yes, that is the path I would use.  Yes, we would need instructions for how to install this.  It should include something like "Install and configure Redis and actioncable" (assuming we're not doing Solid + Rails 8).